### PR TITLE
HLR | Finish up code coverage updates

### PR DESCRIPTION
**Pull request by Mottie**
## Summary

- _(Summarize the changes that have been made to the platform)_
  > Update containers, actions & validation unit tests to bring them up to 100% coverage
  > Minor code cleanup - rename NOD chapter & remove comment
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
- _(Which team do you work for, does your team own the maintenance of this component?)_
  > Benefits decision reviews
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)

[#68890](https://github.com/department-of-veterans-affairs/va.gov-team/issues/68890)

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|  | After |
| ------ | ----- |
| Before | Screenshot 2023-12-07 at 12 37 59 PM | 
| | Screenshot 2023-12-07 at 12 38 08 PM |
| | Screenshot 2023-12-07 at 12 38 23 PM |
| After | Screenshot 2023-12-07 at 8 24 08 AM |
| | Screenshot 2023-12-07 at 12 35 57 PM |
| | Screenshot 2023-12-07 at 12 36 30 PM |

## What areas of the site does it impact?

HLR

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/appeals/996/actions/index.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ export const getContestableIssues = props => {
     if (!foundBenefitType || !foundBenefitType?.isSupported) {
       return Promise.reject({
         error: 'invalidBenefitType',
-        type: foundBenefitType?.label || benefitType || 'Unknown',
+        type: foundBenefitType?.label || benefitType,
```
Comment (Mottie) [inline src/applications/appeals/996/actions/index.js:27]: `benefitType` already has a fallback to `DEFAULT_BENEFIT_TYPE`

File: src/applications/appeals/996/containers/Form0996App.jsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ export const Form0996App = ({
   router,
   savedForms,
   getContestableIssues,
-  contestableIssues = {},
+  contestableIssues,
```
Comment (Mottie) [inline src/applications/appeals/996/containers/Form0996App.jsx:43]: Already has a fallback in `mapStateToProps`

File: src/applications/appeals/996/tests/actions/actions.unit.spec.js [hunk 3]
```diff
@@ -32,10 +32,9 @@ describe('fetch contestable issues action', () => {
 
   it('should dispatch an add person failed action', () => {
     const mockData = { data: 'asdf' };
-    const props = { benefitType: 'compensation' };
     mockApiRequest(mockData, false);
     const dispatch = sinon.spy();
-    return getContestableIssues(props)(dispatch).then(() => {
+    return getContestableIssues()(dispatch).then(() => {
```
Comment (Mottie) [inline src/applications/appeals/996/tests/actions/actions.unit.spec.js:37]: Removing props tests the fallback & increases code coverage

File: src/applications/appeals/996/tests/containers/ConfirmationPage.unit.spec.jsx [hunk 4]
```diff
@@ -91,4 +91,26 @@ describe('Confirmation page', () => {
     expect(tree.find('li .dd-privacy-hidden').length).to.eq(1);
     tree.unmount();
   });
+
+  it('should render with no data', () => {
```
Comment (rocio-desantiago) [inline src/applications/appeals/996/tests/containers/ConfirmationPage.unit.spec.jsx:95]: Oh is this because the data goes away once it's submitted?

**Review by rocio-desantiago — APPROVED** [review]

File: src/applications/appeals/996/tests/containers/ConfirmationPage.unit.spec.jsx (hunk 4 above)
Comment (Mottie) [reply src/applications/appeals/996/tests/containers/ConfirmationPage.unit.spec.jsx:95]: No, this is testing all the branches in the code. If you reload the confirmation page, there won't be any data, but it should still render

Comment (rocio-desantiago) [reply src/applications/appeals/996/tests/containers/ConfirmationPage.unit.spec.jsx:95]: Ah got it. TY

**Pull request merged**